README.md

```diff
@@ -48,47 +82,121 @@ and reference it in settings.py:
 AUTH_USER_MODEL = 'users.User'
 ```
 
-### Adding django-helusers Django apps
+## Optional features
 
-Django-helusers provides two Django apps: `HelusersConfig` provides the
-models and templates needed for helusers to work and `HelusersAdminConfig`
-reconfigures Django admin to work with helusers. The latter includes adding
-a Tunnistamo login button to the admin login screen.
+### Django REST Framework API authentication using JWT
 
-Additionally `social_django` app is needed for the underlying python-social-auth.
+If you have a REST API implemented using Django REST Framework and you want to authorize access to your API using JWTs, then this might be useful to you.
 
-Before adding the apps, you will need to remove `django.contrib.admin`, as
-`HelusersAdminConfig` is implementation of same functionality. You will get
-`django.core.exceptions.ImproperlyConfigured: Application labels aren't unique, duplicates:
-admin`-error, if you forget this step.
+API token authentication is a stateless authentication method, where every request is
+authenticated by checking the signature of the included JWT token. It still
+creates a persistent Django user, which is updated with the information
+from the token with every request.
 
-Then proceed by adding these apps to your `INSTALLED_APPS` in settings.py:
+- Include `drf-oidc-auth` in your project's dependencies.
+- Configure REST framework to use the `ApiTokenAuthentication` class in `settings.py`:
 
 ```python
-INSTALLED_APPS = (
-    'helusers.apps.HelusersConfig',
-    'helusers.apps.HelusersAdminConfig',
-    ...
-    'social_django',
-    ...
-)
+REST_FRAMEWORK = {
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        'helusers.oidc.ApiTokenAuthentication',
+    ),
+}
 ```
 
-Us usual with `INSTALLED_APPS`, ordering matters. `HelusersConfig` must come
-before `HelusersAdminConfig` and anything else providing admin templates.
-Unless, of course, you wish to override the admin templates provided here.
+- Set your deployment-specific variables in `local_settings.py`, e.g.:
```

Comment:
We probably want to encourage using environment variable, but that is not for this PR.

Reply:
Make a ticket